java/src/jmri/jmrit/logixng/implementation/DefaultLogixNGManager.java

```diff
@@ -246,6 +246,21 @@ public void activateAllLogixNGs(boolean runDelayed, boolean runOnSeparateThread)
         // This may take a long time so it must not be done on the GUI thread.
         // Therefore we create a new thread for this task.
         Runnable runnable = () -> {
+
+            // Activate and execute the initialization LogixNGs first.
```

Comment:
Should say global variable initialization.  Change not required.

Reply:
Thanks. Fixed.